backend/controllers/homepage/homepage.js

```diff
@@ -0,0 +1,165 @@
+const {
+  applications: { licences, manageaccounts, moic, pecs },
+  apis: { omic, useOfForce, pathfinder, categorisation, soc },
+} = require('../../config')
+
+const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig }) => [
```

Comment:
Potential for improving this I think (calculating `enabled` along with the roles etc) but couldn't quite get it right so went with this method for now.

Reply:
Yeah I agree it would be nice to turn enabled into a function and let that decide, keeping all the logic in one place. Also they might be good candidates for class types. Happy to help with this in later PR's if you like?